packages/go_router/test/go_router_delegate_test.dart

```diff
@@ -78,6 +78,52 @@ void main() {
     );
   });
 
+  group('replace', () {
```

Comment:
consider adding a test for `replaceNamed` too?

Reply:
I added a test for `replaceNamed` in https://github.com/flutter/packages/pull/2306/commits/44e35ef220cb98e5b34d89bcd2d7c7c3083b3426 Tell me what you think about it